### PR TITLE
add module and deploy docs in Chinese

### PR DESCRIPTION
**Pull request by guoquanwu**
**What does this PR do?**

add module and deploy docs in Chinese

### CODE REVIEW

File: docs/模块化介绍.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+## 模块化的java-tron架构介绍
+![avatar](https://github.com/tronprotocol/java-tron/tree/develop/docs/images/module.png)
+
+模块化后的java-tron目前分为6个模块：framework、protocol、common、chainbase、consensus、actuator，下面分别简单介绍一下各个模块的作用。
+
+### framework
+framework是java-tron的核心模块，不仅是整个链的入口模块，同时也充当粘合剂的作用将其他模块有机地组织起来，framework模块负责各个模块的初始化、流程的跳转。
+
+### protocol
+对于区块链这种分布式网络，简洁高效的数据交换协议尤为重要，protocol模块定义了与java-tron交互的通信协议，是java-tron实现跨语言跨平台的基础，该模块同时定义了
```
Comment (andelf) [inline docs/模块化介绍.md]: > protocol模块定义了与java-tron交互的通信协议

`与` connects 2 objects. omitting one means connects the self with another object. Should be

>  protocol 模块定义了外界与 java-tron 交互的二进制协议格式。

BTW: `通信协议` is for something like UDP/TCP/HTTP/gRPC, not for protobuf binary format.

File: docs/模块化介绍.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### chainbase
+chainbase模块是数据库层面的抽象，像POW、POS、DPOS这类基于概率性的共识算法不可避免的会以一定的概率发生切链，
+因此chainbase定义了一个支持可回退数据库的接口标准，该接口要求数据库实现状态回滚机制、checkpoint容灾机制等。
+另外chainbase模块具有良好的接口抽象设计，任何满足接口实现的数据库都可以作为区块链的底层存储，赋予开发者更多的灵活性，LevelDB和RocksDB是默认提供的两种具体实现。
+下面简单的为大家介绍一下Chainbase模块中重要的几个实现类和接口：
+1. RevokingDatabase：是数据库容器的接口，用于所有可回退数据库的管理，SnapshotManager是该接口的一个实现
+2. TronStoreWithRevoking：支持可回退的数据库的基类，Chainbase类是它的具体实现
+
+
```
Comment (andelf) [inline docs/模块化介绍.md]: Never leave more than 1 empty lines in a .md file.

File: docs/模块化介绍.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+### consensus
+共识机制是区块链中非常重要的模块，常见的有POW、POS、DPOS、PBFT等，联盟链以及其他一些可信网络中也会采用Paxso、Raft等共识机制，共识的选择需要和业务场景相匹配，比如对共识效率敏感实时游戏类就不适合采用POW，而对实时性要求极高的交易所来说PBFT可能是首选。所以支持可替换的共识将是一个有非常有想象力的创造，同时也是实现特定应用区块链的重要一环，即便像Cosmos SDK这样的明星区块链项目依然也停留在应用层为开发者提供一些自主性，底层的共识依然受限于Tendermint共识。
```
Comment (andelf) [inline docs/模块化介绍.md]: for strict spelling, use: `PoW`, `PoS`, `DPoS`.

Comment (andelf) [reply docs/模块化介绍.md]: misspelled `paxos`.
capitalized `raft`.
capitalized `Tendermint`.

File: docs/模块化介绍.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+consensus模块最终目标是能够让应用开发者能够像配置参数那样简单的切换共识机制。
+
+consensus模块将共识过程抽象成几个重要的部分，定义在ConsensusInterface接口中：
+1. start：启动共识服务，可以自定制启动参数
+2. stop：停止共识服务
+3. receiveBlock：定义接收区块的共识逻辑
+4. validBlock：定义验证区块的共识逻辑
+5. applyBlock：定义处理区块的共识逻辑
```
Comment (andelf) [inline docs/模块化介绍.md]: punctuation that has tighter relation with former English words, use English style punctuation.

FYI: you can use only half-width punctuations in zh-cn technical documents.

File: docs/模块化介绍.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+应用开发者可以通过实现ConsensusInterface接口来定制共识，同时社区也正在探索PBFT和DPOS组合共识的方案，进一步降低区块校验延时来满足对实时性要求更高的场景，届时这个混合共识将成为可替换共识的具体案例。
+
+### actuator
+以太坊初创性的引入了虚拟机并定义了智能合约这种开发范式，但对于一些复杂的应用，智能合约不够灵活且受限于性能，这也是java-tron提供创建应用链的一个原因。
+为此java-tron独立出来了actuator模块，该模块为应用开发者提供一种新的开发范式：可以将应用代码直接植入链中而不再将应用代码跑在虚拟机中。
```
Comment (andelf) [inline docs/模块化介绍.md]: `范式` is for `normal form`(which has an academic meaning) or `paradigm`(which is misused by many).
`方式` is OK.

File: docs/模块化介绍.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+actuator是交易的执行器，可以将应用看成是不同交易类型组成的交易集，每类交易都由对应的actuator负责执行。
```
Comment (andelf) [inline docs/模块化介绍.md]: An application may not be a transaction set.

**Comment by andelf** [comment]
This should be closed.
Make a new PR for <https://github.com/tronprotocol/documentation-zh>.
😄

**Comment by guoquanwu** [comment]
fix some spelling mistakes and markdown format

**Pull request closed without merging**